gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -197,9 +197,10 @@ def get_avro_type_from_bigquery_type_mode(bigquery_type, bigquery_mode):
     return avro_type
 
 def update_bigquery_schema_on_append(schema_fields, output_table):
-  # type: (bool) -> None
+  # type: (bool, str) -> None
   # if table does not exist, do not need to update the schema.
   # TODO (yifangchen): Move the logic into validate().
+  """Update BQ schema by combining existing one with a new one, if possible."""
```

Comment:
nit: docstring is a string that is the first statement in a package, module, class or function. Read more [here](http://go/pystyle#383-functions-and-methods). However, if we have type annotation, the doc string goes below the type annotation.

Reply:
Hmm, kinda weird to have a comment, docstring and then a comment again :/. Done, nonetheless.

Reply:
I didn't see any changes here.

I agree with your point. You can refactor it to something like:
"""Update BQ schema by combining existing one with a new one, if possible.

If table does not exist, do not need to update the schema.
TODO (yifangchen): Move the logic into validate().
"""